### PR TITLE
Google map widget on contact page

### LINKED ISSUE #653: [FRONTEND] Embed google maps on contact page
**Is your feature request related to a problem? Please describe.**

Image

**BEFORE MERGING**

- [ ] Storybooks created where possible
- [ ] Tested with [nock](https://tanstack.com/query/v3/docs/framework/react/guides/testing) if data fetching
- [ ] Tests written for critical interactions
- [ ] PR Reviewed (For non-trivial changes)
- [ ] Changes tested after rebasing on master or merging in master (_hint_: `git fetch origin master:master`, then `git rebase master` or `git merge master`)
- [ ] All required PR checks passing

### PR DESCRIPTION
**Pull request by zlrkw11**
closes #653

### CODE REVIEW

**Comment by zlrkw11** [comment]
image

File: client/src/components/composite/Contact/Contact.tsx [hunk 1]
```diff
@@ -13,6 +13,15 @@ const Contact = ({ items }: IContact) => {
         <div className="flex w-full max-w-[800px] flex-col gap-4 py-4">
           <h2 className="text-dark-blue-100 italic">Contact Us</h2>
           <ContactDetailPanel items={items} />
+          <div className="w-full">
```
Comment (bcho892) [inline client/src/components/composite/Contact/Contact.tsx:16]: nit: could include a comment here on what website you generated this from

**Review by bcho892 — APPROVED** [review]

**Review by bcho892 — CHANGES_REQUESTED** [review]
https://www.google.com/maps/place/University+of+Auckland+Snowsports+Club/@-39.2353871,175.5553919,15z/data=!4m6!3m5!1s0x6d6b01d610042f1d:0x1071f4930186b3c0!8m2!3d-39.2353871!4d175.5553919!16s%2Fg%2F1pzszh6c3?entry=ttu

actually the address doesn't look correct

**Comment by bcho892** [comment]
```html
<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3090.3296850757993!2d175.5528169760526!3d-39.235387071653925!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x6d6b01d610042f1d%3A0x1071f4930186b3c0!2sUniversity%20of%20Auckland%20Snowsports%20Club!5e0!3m2!1sen!2snz!4v1722071878514!5m2!1sen!2snz" width="600" height="450" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
```

**Comment by zlrkw11** [comment]
image
- fixed address

**Review by bcho892 — APPROVED** [review]

**Pull request merged**